### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by PanwarJayant**

### CODE REVIEW

File: .github/workflows/deployment-script.yml
Comment (raj-vlabs) [inline .github/workflows/deployment-script.yml]: Please use --depth=1 to clone only the latest code. Look here:
https://github.com/virtual-labs/exp-atterberg-limits-iiith/blob/main/.github/workflows/deployment-script.yml

File: .vscode/settings.json
Comment (raj-vlabs) [inline .vscode/settings.json]: This file should be put in .gitignore

File: experiment/experiment-name.md
Comment (raj-vlabs) [inline experiment/experiment-name.md]: Should Infrared be one word or Infra Red?

File: experiment/posttest.json
Comment (raj-vlabs) [inline experiment/posttest.json:71]: Is it possible to have explanations to the answers?

File: experiment/theory.md
Comment (raj-vlabs) [inline experiment/theory.md]: The Theory has multiple grammatical mistakes, it needs to be reviewed by an SME

File: experiment/simulation/index.html
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Please fix the indentation of the above few lines.

File: experiment/posttest.json
Comment (raj-vlabs) [inline experiment/posttest.json]: Are pretest and posttest going to stay in version 1?

File: experiment/simulation/index.html
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Canvasjs needs license to be used. Please replace with plotly

File: experiment/simulation/index.html
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Please change the alt attribute to describe the image meaningfully. Web crawlers use the alt attribute to understand an image.

File: experiment/simulation/js/main.js
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: Please take the colors out in a separate variable or file with mnemonic keys and hexcode as values. Please take a look at https://github.com/virtual-labs/exp-atterberg-limits-iiith/blob/main/experiment/simulation/js/data.json

File: experiment/simulation/index.html
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Why do we need jQuery?

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
1. Please ensure to fix indentation on all files.
2. Please see if the pretest and posttest can be converted to the quiz 2.0 format.
3. canvasjs is a licensed library, it cannot be used without a license.
4. Multiple grammatical mistakes in many files. Please get them reviewed by the SME

File: experiment/simulation/index.html
Comment (PanwarJayant) [reply experiment/simulation/index.html]: Replaced with Plotly

File: experiment/simulation/index.html
Comment (PanwarJayant) [reply experiment/simulation/index.html]: Was testing something. Forgot to remove it. Removed now.

File: .vscode/settings.json
Comment (PanwarJayant) [reply .vscode/settings.json]: Removed the unnecessary file.

File: experiment/theory.md
Comment (PanwarJayant) [reply experiment/theory.md]: SME still working on the Theory part. Sent the questions for now. Will be adding those questions to the pretest and the postest.

File: experiment/objective.md
Comment (raj-vlabs) [inline experiment/objective.md]: Move the objectives to the Aim file and remove hte objective.md file

File: experiment/objective.md
Comment (raj-vlabs) [inline experiment/objective.md]: Keep both aims in the same tone. Start both with 'To ...'

**Review by raj-vlabs — CHANGES_REQUESTED** [review]

File: experiment/posttest.json
Comment (raj-vlabs) [inline experiment/posttest.json]: Why are we using this complex way of specifying a subscript?

**Review by raj-vlabs — APPROVED** [review]
Approving with minor comments. Please resolve them before merging.

**Pull request merged**
